### PR TITLE
clean up for RH certification

### PR DESCRIPTION
**Pull request by cheewaio**

### CODE REVIEW

File: deploy/olm-catalog/ibm-helm-repo-operator/3.5.0/ibm-helm-repo-operator.v3.5.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -179,7 +204,7 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: ibm-helm-repo-operator
-                image: quay.io/opencloudio/ibm-helm-repo-operator:v3.5.0
+                image: quay.io/opencloudio/ibm-helm-repo-operator:dev
```
Comment (bitscuit) [inline deploy/olm-catalog/ibm-helm-repo-operator/3.5.0/ibm-helm-repo-operator.v3.5.0.clusterserviceversion.yaml]: shouldn't this be release version?

Comment (cheewaio) [reply deploy/olm-catalog/ibm-helm-repo-operator/3.5.0/ibm-helm-repo-operator.v3.5.0.clusterserviceversion.yaml]: yes..good catch

Comment (cheewaio) [reply deploy/olm-catalog/ibm-helm-repo-operator/3.5.0/ibm-helm-repo-operator.v3.5.0.clusterserviceversion.yaml]: updated

**Pull request merged**